ssl/ssl_local.h

```diff
@@ -1213,7 +1213,8 @@ struct ssl_connection_st {
     struct ssl_st ssl;
     /*
      * protocol version (one of SSL2_VERSION, SSL3_VERSION, TLS1_VERSION,
```

Comment:
Really we should drop `SSL2_VERSION` since this is no longer supported. Also this list seems to omit TLS1_1_VERSION which should be there.